doc/Command-Reference.md

```diff
@@ -7961,8 +7962,305 @@ Go Back To [Beginning of the document](#) or [Beginning of this section](#waterm
+  telemetry       docker-sonic-telemetry       SONiC telemetry package       1.0.0      Built-In
+  ```
+
+**sonic-package-manager repository add**
+
+This command will add a new entry in the package database. *NOTE*: this command requires elevated (root) privileges to run.
+
+- Usage:
+  ```
+  Usage: sonic-package-manager repository add [OPTIONS] NAME REPOSITORY
+
+    Add a new repository to database. Repository in Docker Registry V2.
+
+  Options:
+    --default-reference TEXT  Default installation reference
+    --description TEXT        Optional package entry description
+    --help                    Show this message and exit.
+  ```
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-package-manager repository add \
+    cpu-report azure/sonic-cpu-report --default-reference 1.0.0
+  ```
+
+**sonic-package-manager repository remove**
+
+This command will remove an entry from the package database. The package has to be *Not Installed* in order to be removed from package database. *NOTE*: this command requires elevated (root) privileges to run.
+
+- Usage:
+  ```
+  Usage: sonic-package-manager repository remove [OPTIONS] NAME
+
+    Remove package from database.
+
+  Options:
+    --help  Show this message and exit.
+  ```
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-package-manager repository remove cpu-report
+  ```
+
+**sonic-package-manager install**
+
+This command pulls and installs package on SONiC host. *NOTE*: this command requires elevated (root) privileges to run.
+
+- Usage:
+  ```
+  Usage: sonic-package-manager install [OPTIONS] [PACKAGE_EXPR]
+
+    Install package using [PACKAGE_EXPR] in format "<name>==<version>" or "<name>@<tag|digest>"
+
+  Options:
+    --enable                      Set the default state of the feature to
+                                  enabled and enable feature right after
+                                  installation. NOTE: user needs to execute
+                                  "config save -y" to make this setting
+                                  persistent
+    --default-owner [local|kube]  Default owner configuration setting for a
+                                  feature
+    --from-repository TEXT        Fetch package directly from image registry
+                                  repository NOTE: This argument is mutually
+                                  exclusive with arguments: [package_expr,
+                                  from_tarball].
+    --from-tarball FILE           Fetch package from saved image tarball NOTE:
+                                  This argument is mutually exclusive with
+                                  arguments: [package_expr, from_repository].
+    -f, --force                   Force operation by ignoring failures
+    -y, --yes                     Automatically answer yes on prompts
+    -v, --verbosity LVL           Either CRITICAL, ERROR, WARNING, INFO or DEBUG
```

Comment:
This should mention what the default level is.

Reply:
This help comes from https://pypi.org/project/click-log/ click decorator, I cannot modify it unless I copy and modify its code.